docsets/openFrameworks/docset.json

```diff
@@ -15,7 +15,7 @@
         },
         {
             "version": "0.8.4",
-            "archive": "versions/0.8.4/Sample.tgz",
+            "archive": "versions/0.8.4/openFrameworks.tgz",
```

Comment:
You need to add this archive (`versions/0.8.4/openFrameworks.tgz`) to the repo, or remove the `specific_version` entry completely.